pycytominer/tests/test_annotate.py

```diff
@@ -52,19 +52,38 @@ def test_annotate():
     pd.testing.assert_frame_equal(result, expected_result)
 
 
+def test_annotate_platemap_naming():
+
+    # Test annotate with the same column name in platemap and data.
+    platemap_df = PLATEMAP_DF.copy().rename(columns={"well_position": "Metadata_Well"})
```

Comment:
this is really minor, but can you name `platemap_df` something else more descriptive? (just to increase readability)

Reply:
Fixed.